### DISCUSSION TITLE
网址参数清理组件的清理速度能否更快一点？

### DISCUSSION THREAD

**Discussion by XDcedar** [the1812/Bilibili-Evolved#4204] [功能建议 / Ideas]
我一直在用 [Bilibili AntiBV](https://greasyfork.org/zh-CN/scripts/398499-bilibili-antibv) 插件，不仅能自动转换 BV，还会自动清除无效参数（比如`vd_source`）。在启用这个插件后，我发现打开页面时网址栏完全不会出现 `vd_source` 参数，而 Bilibili-Evolved 的 `网址参数清理` 组件却会先展示参数然后再清除，感觉还有改进空间。不知道组件有没有办法做到完全不出现参数？

另注：我更想使用 Bilibili-Evolved 的组件，因为转换 BV 和清除参数是拆分的两个功能，而且最近 B 站把 URL 的形式改了，以前的链接不会以 `/` 结尾的，现在会了，那个插件会把 `/` 给删了，然后刷新链接后 B 站又会把 `/` 加回去，导致历史记录无谓地多了三四条记录，就很难受

**Comment by the1812** [comment]
组件运行时机就是比较晚的 (需要优先插入样式, 执行插件逻辑之类的), 只有这个设置能稍微往前调一调
<img src="https://github.com/the1812/Bilibili-Evolved/assets/26504152/c8a3452f-e1ff-490f-b784-bfaf9f4349b8" width="300">

**Comment by XDcedar** [comment]
喔，试了一下发现这样调整就可以了，已经用上了，效果还不错！！
